src/helpers/subgraph/position/getPositionProfit.ts

```diff
@@ -0,0 +1,75 @@
+import axios from "axios";
+import { utils } from "ethers";
+const url = "https://api.studio.thegraph.com/query/4540/abra-test/v0.0.21/";
```

Comment:
Awesome that we're finally using the subgraph. This only works on ethereum, right now?

Reply:
that logic shouldn't be in this PR, that's what #117 is for - we took it out of here🙏